content/docs/command-reference/params/index.md

```diff
@@ -115,16 +119,20 @@ The `train.py` script will have some code to parse and load the needed
 parameters. For example:
 
 ```py
-import yaml
+from ruamel.yaml import YAML
```

Comment:
It looks like there's a couple other cases of `import yaml` in `stage add` and `run` docs. Would you mind making similar changes there?

Reply:
It might be nice since it saves a `pip install`, but it might also look like it's required to use the DVC API.

@daavoo Let's use `ruamel.yaml` and make a separate issue to discuss `dvc.utils.serialize`.

Reply:
I thought a little more on this and realized `ruamel.yaml` is a DVC dependency, so it would also be installed. Whether it is better to provide a specific external library or an internal DVC function is a tough question, and neither looks as good as `import yaml` unfortunately.

Reply:
> It might be nice since it saves a `pip install`, but it might also look like it's required to use the DVC API.
> 
> @daavoo Let's use `ruamel.yaml` and make a separate issue to discuss `dvc.utils.serialize`.

It's an internal API, let's not do anything.